sklearn/linear_model/least_angle.py

```diff
@@ -1456,6 +1456,7 @@ def fit(self, X, y, copy_X=True):
 
         R = y[:, np.newaxis] - np.dot(X, coef_path_)  # residuals
         mean_squared_error = np.mean(R ** 2, axis=0)
```

Comment:
here we take the mean of sum of squares, only to multiply by n_samples afterwards. It seems mean_squared_error is not used elsewhere in the local scope, we could save some cycles.